templates/repo/contributors.tmpl

```diff
@@ -0,0 +1,33 @@
+{{template "base/head" .}}
+<div role="main" aria-label="{{.Title}}" class="page-content repository commits">
+	{{template "repo/header" .}}
+	<div class="ui container">
+		<h2 class="ui header">{{DateTime "long" .DateFrom}} - {{DateTime "long" .DateUntil}}
+			<div class="ui right">
+				<!-- Contribution type -->
+				<div class="ui floating dropdown jump filter">
+					<div class="ui basic compact button">
+						<span class="text">
+							{{.locale.Tr "repo.contributors.contribution_type.filter_label"}} <strong>{{.ContributionTypeText}}</strong>
+							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						</span>
+					</div>
+					<div class="menu">
+						<a class="{{if eq .ContributionType "commits"}}active {{end}}item" href="{{$.RepoLink}}/contributors/commits">{{.locale.Tr "repo.contributors.contribution_type.commits"}}</a>
+						<a class="{{if eq .ContributionType "additions"}}active {{end}}item" href="{{$.RepoLink}}/contributors/additions">{{.locale.Tr "repo.contributors.contribution_type.additions"}}</a>
+						<a class="{{if eq .ContributionType "deletions"}}active {{end}}item" href="{{$.RepoLink}}/contributors/deletions">{{.locale.Tr "repo.contributors.contribution_type.deletions"}}</a>
+					</div>
+				</div>
+			</div>
+		</h2>
```

Comment:
I need to move these lines from `tmpl` file to my Vue component in order to fix last 3 issues from [here](https://github.com/go-gitea/gitea/pull/25439#issuecomment-1605712716). How do I use `.locale.Tr`, `DateTime` or `svg` in a Vue template? Can someone help?

Reply:
Also I just realized classes here are Fomantic UI classes. They don't work properly in Vue for some reason :/ For example, the dropdown menu doesn't open when it is written in a Vue file.